readthedocs/projects/migrations/0010_migrate_domain_data.py

```diff
@@ -9,7 +12,7 @@
 if sys.version_info > (3,):
     import urllib.parse as urlparse
 else:
-    import urlparse
+    import urllib.parse
```

Comment:
This block seems redundant, doesn't `future` resolve the need for this sort of import pattern?